plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -237,4 +241,8 @@ export class SonataFlowService {
     }
     return false;
   }
+
+  public getFetchErrorMessage(urlToFetch: string, responseStr: string): string {
+    return `Response status was NOT successful when fetch(${urlToFetch}). Received response ${responseStr}`;
```

Comment:
Still wondering about `fetch(${urlToFetch})`; what if the same URL is used but with different method, this can get confusing.
Similarly, do we want to output the body of a POST that was used when the request fails? It can be dangerous is sensitive information are in the body but it may come in handy to investigate when something goes wrong.
Maybe with DEBUG log level the `fetch` gives more info so let's not include the body, but the method can be useful, i.e: `Response status was NOT successful when $ {method} fetch(${urlToFetch})`

Reply:
Agree. What about it now?